frontend/my-app/src/Pages/LandingPage.tsx

```diff
@@ -167,15 +218,40 @@ const LandingPage: React.FC = () => {
           <Box
             component="span"
             sx={{ p: 5 }}
-          >
-          </Box>
-        <div />
+          />
+        <Button variant="outlined" color="primary" onClick={handleToggle} sx={{ mb: 5 }}>
+            Add Friends
+        </Button>
+        <Dialog
+          open={open}
+          onClose={handleClose}
+          sx={{
+            borderRadius: '20px'
+          }}>
+          <DialogTitle>Add Friend</DialogTitle>
+          <DialogContent>
+            <TextField
+              autoFocus
+              margin="dense"
+              id="name"
+              label="Email Address"
+              type="email"
+              fullWidth
+              variant="standard"
+            />
+          </DialogContent>
+          <DialogActions>
+            <Button onClick={handleClose}>Cancel</Button>
+            <Button onClick={handleSubmit}>Send Request</Button>
```

Comment:
Seems like there are different style here. For example, here we write:
`<Button onClick={handleClose}>Cancel</Button>`
But on @222 it's
`<Button variant="outlined" color="primary" onClick={handleToggle} sx={{ mb: 5 }}>
            Add Friends
</Button>`
We might want to keep things as consistent as possible